From: "Jason St. John" <jstjohn@purdue.edu>
Date: Tue, 17 Dec 2013 18:48:43 -0500
Subject: man: be more explicit about option arguments that take 128-bit IDs
 in journalctl(1)

It may not be immediately obvious to the reader what "ID128" is, so replace the
example option argument "ID128" with "128-bit-ID".

(cherry picked from commit 9a7adc9a0ed53d6e422321e9196eb83d18afcba5)
---
 man/journalctl.xml | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/man/journalctl.xml b/man/journalctl.xml
index d936933..dced90f 100644
--- a/man/journalctl.xml
+++ b/man/journalctl.xml
@@ -725,7 +725,7 @@
 
                         <varlistentry>
                                 <term><option>--list-catalog
-                                <optional><replaceable>ID128...</replaceable></optional>
+                                <optional><replaceable>128-bit-ID...</replaceable></optional>
                                 </option></term>
 
                                 <listitem><para>List the contents of
@@ -734,7 +734,7 @@
                                 description strings.</para>
 
                                 <para>If any
-                                <replaceable>ID128</replaceable>s are
+                                <replaceable>128-bit-ID</replaceable>s are
                                 specified, only those entries are shown.
                                 </para>
                                 </listitem>
@@ -742,7 +742,7 @@
 
                         <varlistentry>
                                 <term><option>--dump-catalog
-                                <optional><replaceable>ID128...</replaceable></optional>
+                                <optional><replaceable>128-bit-ID...</replaceable></optional>
                                 </option></term>
 
                                 <listitem><para>Show the contents of
@@ -753,7 +753,7 @@
                                 files.</para>
 
                                 <para>If any
-                                <replaceable>ID128</replaceable>s are
+                                <replaceable>128-bit-ID</replaceable>s are
                                 specified, only those entries are shown.
                                 </para>
                                 </listitem>
